packages/did/src/Did.signature.ts

```diff
@@ -99,3 +98,18 @@ export function isDidSignature(
     return false
   }
 }
+
+/**
+ * Transforms the output of a [[SignCallback]] into the [[DidSignature]] format suitable for json-based data exchange.
+ *
+ * @param input Signature data returned from the [[SignCallback]].
+ * @param input.signature Signature bytes.
+ * @param input.keyUri DID URI of the key used for signing.
+ * @returns A [[DidSignature]] object where signature is hex-encoded.
+ */
+export function signatureToJson({
+  signature,
+  keyUri,
+}: SignResponseData): DidSignature {
+  return { signature: Crypto.u8aToHex(signature), keyUri }
```

Comment:
Also here, I think the raw stuff should all be byte arrays, and hex conversion should only ever be needed when serialising to, e.g., print.

Reply:
I think we're on the same page here, but this function is built explicitly for this serialisation use case. Our interfaces like credentials, quotes and so on are data formats that are designed with serialisation in mind, so I don't think Uint8Arrays work there.

Reply:
Alright, then sounds good 👍